x/oracle/keeper/keeper.go

```diff
@@ -67,8 +67,8 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
 
-// GetExchangeRate gets the consensus exchange rate of USD denominated in the
-// denom asset from the store.
+// GetExchangeRate gets the consensus exchange rate of an asset
```

Comment:
The former comment was more correct. It's the exchange rate against the USD denomination.